share/spice/product_hunt/product_hunt.css

```diff
@@ -20,7 +20,11 @@
     margin-bottom: 10px;
 }
 
-.is-mobile .tile--producthunt .tile__content {
+.tile--producthunt .tile__title {
+    height: 1em;
+}
+
+.is-mobile .tile__content {
```

Comment:
I think we can remove all this now -- the variants and internal CSS should be taking care of the heights/spacing :)